components/Notification/Notification.tsx

```diff
@@ -0,0 +1,18 @@
+interface NotificationProps {
+  children: JSX.Element;
+  hasIcon?: boolean;
+}
+export const Notification = ({
+  children,
+  hasIcon = false,
+}: NotificationProps) => {
+  return (
+    <div
+      className={`flex justify-between bg-turquoise/[0.1] rounded-lg py-8 pr-[83px] ${
+        hasIcon ? " pl-[7px]" : " pl-[83px]"
+      }`}
```

Comment:
```suggestion
      className={`flex bg-turquoise/[0.1] rounded-lg py-8 pr-[83px] ${
        hasIcon ? "pl-[7px] justify-between" : "justify-center"
      }`}
```

Acá se podría usar un justify-center en vez de un padding left. Además el espacio antes de los estilos no es necesario ya que ya hay un espacio antes del template literal